api/apiFunctions.test.tsx

```diff
@@ -0,0 +1,43 @@
+ import { loginAccount, logoutAccount } from './apiFunctions';
+import { account } from './config';
+
+describe('Auth Functions', () => {
+
+  jest.mock('./apiFunctions', () => ({
+    loginAccount: jest.fn()
+  }))
+
+  describe('login account successful', () => {
+    it('should show user login successfully', async () => {
+      const userDummy = { 
+          email: "testemail@email.com",
+          password: "test1234",
+        }
+     await loginAccount(userDummy);
```

Comment:
Please check with @ryandotfurrer on how to setup prettier. I keep seeing your code formatting way off.